(2023.01.11) Stefan Verkoyen (sverkoyen.contractor@libertyglobal.com)
ONEM-28788 Integrate latest Rialto versions in OneMW

Index: a/src/third_party/starboard/rdk/shared/BUILD.gn
===================================================================
--- a/src/third_party/starboard/rdk/shared/BUILD.gn
+++ b/src/third_party/starboard/rdk/shared/BUILD.gn
@@ -78,8 +78,8 @@ if (rdk_enable_securityagent) {
 }
 
 if (rdk_enable_ocdm) {
-  pkg_config("ocdm") {
-    packages = [ "ocdm" ]
+  pkg_config("ocdmRialto") {
+    packages = [ "ocdmRialto" ]
     defines = [ "HAS_OCDM=1" ]
   }
 }
@@ -481,7 +481,7 @@ static_library("starboard_platform") {
   }
 
   if (rdk_enable_ocdm) {
-    configs += [ ":ocdm" ]
+    configs += [ ":ocdmRialto" ]
   }
 
   public_deps = [
